arm-relay/2017-04-01/swagger/relay.json

```diff
@@ -1842,8 +1837,18 @@
       "properties": {
         "provisioningState": {
           "type": "string",
-          "readOnly": true,
-          "description": "Provisioning state of the namespace. supported value - Created/Succeeded/Deleted/Failed/Unknown"
+          "enum": [
+            "Created",
+            "Succeeded",
+            "Deleted",
+            "Failed",
+            "Updating",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "provisioningStateEnum",
+            "modelAsString": false
```

Comment:
should this be true? Making it "false" will make this an enum, if there are other possible values supported by the service now or later, this may fail/cause breaking changes in the SDK. Could you confirm you want to make this a real enum?